packages/terra-dialog-modal/src/terra-dev-site/test/dialog-modal/DialogModalWithCustomHeaderAndCustomFooter.test.jsx

```diff
@@ -3,8 +3,6 @@ import Button from 'terra-button';
 import DialogModal from '../../../DialogModal';
 
 const CustomHeaderFooterStyles = {
-  border: '2px dashed #4e832b',
-  color: '#4e832b',
```

Comment:
These custom colors can cause accessibility violations in downstream theming libraries consuming these tests.